lib/postgres_ext/arel/5.0/predications.rb

```diff
@@ -0,0 +1,39 @@
+require 'arel/predications'
```

Comment:
Exactly, I was following the structure previously set, but once I have this working I will refactor as cleanly as possible.